lib/rules/no-else-return.js

```diff
@@ -189,13 +198,14 @@ module.exports = {
             return checkForReturnOrIf(node);
         }
 
+
         /**
-         * Check the if statement
+         * Check the if statement, but don't catch else-if blocks.
          * @returns {void}
          * @param {Node} node The node for the if statement to check
          * @private
          */
-        function IfStatement(node) {
+        function LegacyIfStatement(node) {
             const parent = context.getAncestors().pop();
```

Comment:
Can this line be replaced with `const parent = node.parent`? (I realize the original version was here before this change.)

Reply:
giving it a go.